### PR TITLE
Adding Android 10 support

### PR DESCRIPTION
**Pull request by pcrepieux**
- provides an agent to manage touch events. If minitouch can no longer manager inputs, it can forward events to this agent that will use Android InputManager.

- wraps SecurityException when getting ids that are not accessible anymore.

- defines a notification channel and adds permission for the Service

- bumps libs/plugins/apk version numbers

### CODE REVIEW

**Comment by pcrepieux** [comment]
See also related PR for minitouch: https://github.com/openstf/minitouch/pull/50
I've done it that way to keep the existing components, at least as a first step.

**Comment by histronger** [comment]
Cool! I'll try it. Thank you for your work!

**Comment by pcrepieux** [comment]
Great, hope you'll have it working smoothly.

File: app/src/main/java/jp/co/cyberagent/stf/Service.java [hunk 1]
```diff
@@ -92,8 +108,15 @@ public void onCreate() {
 
         clipboardManager = getSystemService(Context.CLIPBOARD_SERVICE);
 
+        String channelId;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            channelId = createNotificationChannel("stf_channel", "STF Channel");
+        } else {
+           channelId="";
```
Comment (koral--) [inline app/src/main/java/jp/co/cyberagent/stf/Service.java]: Could you reformat the code to use consistent spacing, wrapping etc?

**Review by koral-- — COMMENTED** [review]
Apart from code formatting LGTM

File: app/src/main/java/jp/co/cyberagent/stf/Service.java (hunk 1 above)
Comment (pcrepieux) [reply app/src/main/java/jp/co/cyberagent/stf/Service.java]: Sure 👍 Done

**Comment by pcrepieux** [comment]
Recent tests with different devices showed up an issue with the up event not properly managed. This is the purpose of [commit bd73070](https://github.com/openstf/STFService.apk/pull/38/commits/bd730709265b05f36ad4f3c649335064e9e30ca3)

**Review by koral-- — APPROVED** [review]

**Pull request merged**